x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/NoArgumentsConfigurationFunction.java

```diff
@@ -17,14 +17,14 @@
 import java.util.List;
 import java.util.Objects;
 
-public abstract class ConfigurationFunction extends ScalarFunction {
+import static java.util.Collections.emptyList;
+
+public abstract class NoArgumentsConfigurationFunction extends ConfigurationFunction {
```

Comment:
The name is unfortunate - how about `SqlConfigurationFunction`, `BasicConfigurationFunction` or `DefaultConfigurationFunction`, etc...

Reply:
Agree. Will change.